Add support for user group resources (#473)
spacelift-io/terraform-provider-spacelift: docs/resources/idp_group_mapping.md

---
# generated by https://github.com/hashicorp/terraform-plugin-docs
page_title: "spacelift_idp_group_mapping Resource - terraform-provider-spacelift"
subcategory: ""
description: |-

>   spacelift_idp_group_mapping represents a mapping (binding) between a user group (as provided by IdP) and a Spacelift User Management Policy. If you assign permissions (a Policy) to a user group, all users in the group will have those permissions unless the user's permissions are higher than the group's permissions.

Reviewer: I wouldn't use Spacelift User Management Policy.
```suggestion
  spacelift_idp_group_mapping represents a mapping (binding) between a user group (as provided by IdP) and a set of access management rules. All users in the group will have those permissions unless the user's permissions are higher than the group's permissions.
```
